core/src/main/java/com/yahoo/ycsb/Client.java

```diff
@@ -553,6 +668,20 @@ private static void exportMeasurements(Properties props, int opcount, long runti
       exporter.write("OVERALL", "RunTime(ms)", runtime);
       double throughput = 1000.0 * (opcount) / (runtime);
       exporter.write("OVERALL", "Throughput(ops/sec)", throughput);
+      
+      boolean trackJVMStats = props.getProperty(Measurements.MEASUREMENT_TRACK_JVM_PROPERTY, 
```

Comment:
Instead of having two places to parse this property, could we just ask the status thread if it's been collecting JVM stats?